src/main/kotlin/org/rust/ide/typing/paste/RsConvertJsonToStructCopyPasteProcessor.kt

```diff
@@ -111,6 +112,26 @@ class RsConvertJsonToStructCopyPasteProcessor : CopyPastePostProcessor<TextBlock
     }
 }
 
+@TestOnly
+var CONVERT_JSON_SERDE_PRESENT: Boolean = false
+
+fun convertJsonWithSerdePresent(hasSerde: Boolean, action: () -> Unit) {
+    val original = CONVERT_JSON_SERDE_PRESENT
+    CONVERT_JSON_SERDE_PRESENT = hasSerde
+    try {
+        action()
+    } finally {
+        CONVERT_JSON_SERDE_PRESENT = original
+    }
+}
```

Comment:
I think it's better to make `CONVERT_JSON_SERDE_PRESENT` private, drop `@TestOnly` from it and add the annotation to `convertJsonWithSerdePresent ` since `convertJsonWithSerdePresent ` is supposed to be used only in tests but `CONVERT_JSON_SERDE_PRESENT` is technically used in production code

Reply:
Ah, of course, you're right. I'll change it in my other JSON conversion PR.